keda/templates/13-keda-poddisruptionbudget.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.podDisruptionBudget }}
```

Comment:
This is a breaking change, can you make it non-breaking please?

Reply:
Any hint how i could do that without breaking?

Reply:
Use `podDisruptionBudget` when specified, otherwise use new value